buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes

```diff
@@ -0,0 +1,238 @@
+                                            rm -rf /tmp/javasharedresources /temp/javasharedresources ${buildWorkspace}/../../javasharedresources
+                                        """
+                                    }
+
+                                    // Cleanup OSX shared memory
+                                    if (nodeLabels.contains('sw.os.osx')) {
+                                        retry(2) {
+                                            sh """
+                                                ipcs -ma
+                                                ipcs -ma | awk '/^m / { if (\$9 == 0) { print \$2 }}' | xargs -n 1 ipcrm -m
+                                                ipcs -ma
+                                            """
+                                        }
+                                    }
+
+                                    // Clean up defunct pipelines workspaces
+                                    dir("${buildWorkspace}/../") {
+                                        retry(3) {
+                                            if (nodeLabels.contains('sw.os.aix')) {
+                                                sh "rm -rf Build* Test* PullRequest*"
+                                                sh "ls -al"
+                                            } else {
+                                                sh "ls | grep -v ${JOB_NAME} | xargs rm -rf"
+                                            }
+                                            sh "sleep 1m"
+                                        }
+                                    }
+                                }
+                            }
+
+                            if (MODES.contains('sanitize')) {
+                                stage("${nodeName} - Sanitize slave") {
+                                    // list user procs
+                                    sh "hostname && ps -f -u ${env.USER} | grep -EIv ${JOB_NAME} | grep -Eiv 'slave.jar' | grep -Eiv 'remoting.jar'| grep -Eiv '${env.USER}@notty' | sort"
+                                    // kill procs and reconnect slave
+                                    sanitize_node(nodeName)
+                                }
+                            }
+                        }
+                    }
+                }
+
+                if (offlineSlaves) {
+                    println("Offline slaves: ${offlineSlaves.toString()}")
+                }
+
+                if (unavailableSlaves) {
+                    println("Unavailable slaves: ${unavailableSlaves.toString()}")
+                }
+
+                parallel jobs
+
+            } catch(e) {
+                //slackSend channel: '#jenkins', color: 'danger', message: "Failed: ${env.JOB_NAME} #${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
+                //slackSend channel: '@AdamBrousseau', color: 'danger', message: "Failed: ${env.JOB_NAME} #${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
```

Comment:
Should remove this line

Reply:
will parameterize slack channel and send notification if set